src/cephadm/cephadm

```diff
@@ -7401,6 +7405,11 @@ def _get_parser():
     parser_pull = subparsers.add_parser(
         'pull', help='pull latest image version')
     parser_pull.set_defaults(func=command_pull)
+    parser_pull.add_argument(
+        '--insecure',
+        action='store_true',
+        help='Assume insecure (non-TLS) registry'
```

Comment:
Could you please either mention that this is for development purposes only or use `parser.add_argument('--foo', help=argparse.SUPPRESS)` ?